Base.rte/Scripts/Shared/FlippedFrame.lua

```diff
@@ -1,4 +1,6 @@
 --Use this script to display a different sprite frame when facing left, instead of the same sprite mirrored
+function Create(self)
```

Comment:
Kill this off. You shouldn't need this on dev, if you still do, message me cause that's not good!